mono/mini/genmdesc.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
```

Comment:
on some linux distributions (e.g. Arch Linux) this would get you python3, thus either

1. we make sure it prints a warning when using python3 (http://sweetme.at/2013/10/21/how-to-detect-python-2-vs-3-in-your-python-script/)
2. or, we make sure it runs on python3. (as of now, there are some `print` usages in the script that aren't python3 compatible).